app/controllers/dashboard/dashboard-activity-controller.js

```diff
@@ -13,13 +13,59 @@ const CORRELATION_HEADER = require('../../utils/correlation_header').CORRELATION
 const ConnectorClient = require('../../services/clients/connector_client').ConnectorClient
 const { isADirectDebitAccount } = require('../../services/clients/direct_debit_connector_client.js')
 const auth = require('../../services/auth_service.js')
-const connectorClient = () => new ConnectorClient(process.env.CONNECTOR_URL)
 const { datetime } = require('@govuk-pay/pay-js-commons').nunjucksFilters
+const goLiveStage = require('../../models/go-live-stage')
+
+const connectorClient = () => new ConnectorClient(process.env.CONNECTOR_URL)
 const getTimespanDays = (fromDateTime, toDateTime) => moment(toDateTime).diff(moment(fromDateTime), 'days')
 
 // Constants
 const clsXrayConfig = require('../../../config/xray-cls')
 
+const getLinksToDisplay = (service, account) => {
```

Comment:
```suggestion
const getLinksToDisplay = function getLinksToDisplay (service, account) => {
```

It would be useful to name these instead of using anonymous functions, if an exception is thrown the name will then be used in the stack trace in favour of `anonymous`.

Reply:
Thanks, I've made this change